### DISCUSSION TITLE
How to apply resources, if no App.xaml present?

### DISCUSSION THREAD

**Discussion by MatthiasSchilder** [Kinnara/ModernWpf#453] [General]
I have a class library containing a window I want to style with ModernWpfUI. Because it is a class library, and the library is used from a Windows Form application, I don't have an App.xaml anywhere. I've tried to put it in my Windows.Resources, but this gives me a broken appearance at runtime (see screenshot). Is it possible to apply the resources, if there is no App.xaml present?

```
<Window [...]
      ui:WindowHelper.UseModernWindowStyle="True">
[...]>
<Window.Resources>
	<ResourceDictionary>
		<ResourceDictionary.MergedDictionaries>
			<ui:ThemeResources />
			<ui:XamlControlsResources />
		</ResourceDictionary.MergedDictionaries>
		<converters:DauerstufeWiederkehrzeitConcatConverter x:Key="MultiBindingConcatConverter" />
	</ResourceDictionary>
</Window.Resources>
```

![image](https://user-images.githubusercontent.com/313898/158780409-c1a77de4-b190-4551-ad04-3f8f5efcd023.png)

**Comment by luojunyuan** [comment]
Add a `Properties/DesignTimeResources.xaml` file at your library.
The file looks like
```xml
<ResourceDictionary
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:ui="http://schemas.modernwpf.com/2019"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
    <ResourceDictionary.MergedDictionaries>
        <ui:ThemeResources />
        <ui:XamlControlsResources />
    </ResourceDictionary.MergedDictionaries>
</ResourceDictionary>
```
![image](https://user-images.githubusercontent.com/28323948/160600567-526671d7-c785-4c25-901a-5ddf6c75af2f.png)
